CorsixTH/Lua/endconditions.lua

```diff
@@ -32,13 +32,15 @@ local local_criteria_variable = {
     -- New criteria
   {name = "staff_happiness",  icon = 16, icon_file = "MPointer", formats = 2, two_tooltips = true},
   {name = "patient_happiness",icon = 18, icon_file = "MPointer", formats = 2, two_tooltips = true},
+  {name = "year",             icon = 58, icon_file = "Font04V",  formats = 4, two_tooltips = true}, -- Starts from 0 not 2000
 }
 
 -- A table of functions for fetching criteria values that cannot be measured
 --   directly from a hospital attribute of the same name.
```

Comment:
With your addition, this comment is now false.

While at it, remove the weird indentation at the 2nd line.